CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
   # Any Clang or any GCC
   add_compile_options(
     -Wno-multichar
-
+    -Wno-format-truncation # squelch warning about snprintf truncating strings (see PR #3977)
```

Comment:
You need this at line 73 in the gcc block; this is gcc+clang both block

Reply:
Oops, thanks for that